src/components/Guilds/ProposalPage/ProposalVoteCard/index.tsx

```diff
@@ -164,6 +170,14 @@ const ProposalVoteCard = () => {
           </ButtonsContainer>
         )}
       </SidebarCardContent>
+      <VoteConfirmationModal
+        isOpen={modalOpen}
+        onDismiss={() => setModalOpen(false)}
+        onConfirm={confirmVoteProposal}
+        selectedAction="Yes"
+        votingPower="0.12"
+        previousVotingPercentage="0"

```

Comment:
can we make this more general?

Reply:
What do you mean with "more general"? This is hardcoded. Dealing with real values on next pr.